docs/stock/understanding-highcharts-stock.md

```diff
@@ -5,30 +5,92 @@ Highcharts Stock is based on Highcharts, meaning it has all the core functionali
 
 ![understanding_highstock.png](understanding_highstock.png)
 
-Highcharts Stock also supports candlestick and OHLC charts.
+Highcharts Stock also supports various financial series types.
 
-Navigator
+<iframe style="width: 100%; height: 550px; border: none;" src=https://highcharts.github.io/highcharts-utils/samples/#gh/352ea24215ec4f79bb50ead0152430087bc3551e/sample/stock/interactive-docs/series-type allow="fullscreen"></iframe>
+
+```js
+chart.series[0].update({
+    type: 'candlestick'
+});
+```
+
+See [Update series method](https://api.highcharts.com/class-reference/Highcharts.Series#update) and [series API options](https://api.highcharts.com/highstock/series).
+
+Navigator and scrollbar
 ---------
 
-Allows you to fine tune the range of the chart which is displayed.
+Allows you to fine tune the range of the chart which is displayed and scroll through it.
+
+See [Navigator](https://highcharts.com/docs/stock/navigator) for more information.
+
+<iframe style="width: 100%; height: 640px; border: none;" src=https://highcharts.github.io/highcharts-utils/samples/#gh/352ea24215ec4f79bb50ead0152430087bc3551e/sample/stock/interactive-docs/navigator allow="fullscreen"></iframe>
+
+```js
+chart.update({
+    navigator: {
+        enabled: true,
+        height: 100
+    },
+    scrollbar: {
+        enabled: false
+    }
+});
+```
+To update navigator and scrollbar use [Chart update](https://api.highcharts.com/class-reference/Highcharts.Chart#update). See [navigator API options](https://api.highcharts.com/highstock/navigator).
 
-See [Navigator](https://highcharts.com/docs/stock/navigator) for more information.
 
 Range selector
 --------------
 
 Allows you to quickly select a range to be shown on the chart or specify the exact interval to be shown.
 
+<iframe style="width: 100%; height: 600px; border: none;" src=https://highcharts.github.io/highcharts-utils/samples/#gh/352ea24215ec4f79bb50ead0152430087bc3551e/sample/stock/interactive-docs/range-selector allow="fullscreen"></iframe>
```

Comment:
1. The buttons update always updates buttons only once. They always stays `Month - 6 Months -  Week - Year`. Should we also change the order to Week - Month - 6 Months - Year? 

2. How about adding one more button: position. So RS will be rendered on top, or below the navigator?

3. In addition, there seems to be a bug with reserved space for buttons (when we disable buttons, there is extra space at top). Did you check if it's already fixed in the master (or update refactor by Pueblo) branch?

Reply:
1. The link over here is leading to an old commit with previous demo. Up-to-date demos are on #21537 (now I think I made it too confusing, sorry)

2. Good idea, I'll add it to demos

3. Will check 🫡

Reply:
3. If we are talking about the issue, where after disable -> enable, chart does not give RS extra margin (so it's rendered below yAxis gridLine) then this issue is visible on master and Pueblo's branch sadly

https://jsfiddle.net/BlackLabel/57cfbtnw/